cypress/fixtures/objects/validationObjects/account.json

```diff
@@ -19,6 +19,7 @@
                 "type": "regEx",
                 "validations": [
                     {
+                        "mode": "regex",
```

Comment:
why are we having a **mode : regex** if we have **type : regex** at the top of the json